corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -419,20 +419,23 @@ <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
   <legend aria-hidden="true" class="sr-only">{% trans "Answer Choices" %}</legend>
     <div class="sel clear">
       <div data-bind="foreach: choices, as: 'choice'">
-        <div data-bind="css: { checkbox: $parent.isMulti, radio: !$parent.isMulti }">
+        <div data-bind="css: { checkbox: $parent.isMulti, radio: !$parent.isMulti }, addColStyle: $parent.hideLabel">
```

Comment:
>  I tried using the traditional, simple attr binding to set the class, but when hideLabel is false and the class attribute is null it still overwrites Bootstrap classes, causing all select menus to lose some of their special formattings.

Was it this checkbox/radio class that was getting blanked out, or more than that? [These docs](https://knockoutjs.com/documentation/css-binding.html) make me think you could use `class` for the col class and it wouldn't interfere with the css binding.

Reply:
You're comletely on point here, it was just the CSS bindings. I swear I'd tried using just `class` instead of `'class'` inside of `attr` but I must have done something wrong because I just tried again and it seems to work great. Yay, no custom binding!